docs/guide/test-context.md

```diff
@@ -115,15 +115,17 @@ const myTest = test.extend({
 })
 
 // todosFn will not run
-myTest('', () => {}) // no fixture is available
-myTets('', ({ archive }) => {}) // only archive is available
+myTest('', () => {})
+myTets('', ({ archive }) => {})
 
 // todosFn will run
-myTest('', ({ todos }) => {}) // only todos is available
-myTest('', (context) => {}) // both are available
-myTest('', ({ archive, ...rest }) => {}) // both are available
+myTest('', ({ todos }) => {})
 ```
 
+::: warning
+When using `test.extend()`, you should always use the object destructuring pattern `{ todos }` to access context both in fixture function and test function.
```

Comment:
I think we should allow normal access (without destructuring) if there are no fixtures added with `test.extend` for backwards compatibility

Reply:
Make sense